process/wg_readme_template.md

```diff
@@ -0,0 +1,46 @@
+_This document is intended to be a template for working groups established by the TAC, to be used as the README.md file in it's repository. All items in italics are descriptions of what context should be there; it is intended to be replaced by content. It is modeled off the [CNCF Serverless WG](https://github.com/cncf/wg-serverless)._
+
+_The intention is that each WG has it's own GitHub repo under the [ASWF organization](https://github.com/AcademySoftwareFoundation)._
+
+# _Working group name_
+
+_Describe the purpose of the group in no more that 4-5 sentences._
+
+Goals of the WG are:
+
+1. _goal 1_
+2. ...
+
+Non-goals of the WG are:
+
+1. _non goal 1_
+2. ...
+
+The TAC sponsor of this working group is _TAC sponsor name_
```

Comment:
maybe "The TAC member sponsor" to make it clear that WGs need to be sponsored by a specific member?